docs/site/HasMany-relation.md

```diff
@@ -529,9 +531,107 @@ export class CustomerRepository extends DefaultCrudRepository {
   for a certain relation. e.g
   `customerRepository.inclusionResolvers.delete('orders')`
 
-{% include note.html content="
-Inclusion with custom scope:
-Besides specifying the relation name to include, it's also possible to specify additional scope constraints.
-However, this feature is not supported yet. Check our GitHub issue for more information:
-[Include related models with a custom scope](https://github.com/strongloop/loopback-next/issues/3453).
-" %}
+### More use cases - query multiple relations:
+
+It is possible to query several relations or nested include relations with
+custom scope. Once you have the inclusion resolver of each relation set up, the
+following queries would allow you traverse data differently:
+
+In our example, we have relations:
+
+- `Customer` _belongsTo_ an `Address` - denoted as `address`.
```

Comment:
I find this relation weird, it's usually an address that belongs to a customer. Have you considered using `HasOne` or `HasMany` relation instead?